apps/agreement/contracts/Agreement.sol

```diff
@@ -112,37 +111,37 @@ contract Agreement is ILockManager, IAgreement, IArbitrable, IACLOracle, AragonA
         uint256 disputeId;                       // Identification number of the dispute on the arbitrator
-        uint256 ruling;                          // Ruling given for the action's dispute
+        uint256 ruling;                          // Ruling given from the arbitrator for the dispute
     }
 
     struct CollateralRequirement {
         ERC20 token;                        // ERC20 token to be used for collateral
         uint64 challengeDuration;           // Challenge duration in seconds, during which the submitter can raise a dispute
         uint256 actionAmount;               // Amount of collateral token to be locked from the submitter's staking pool when creating actions
         uint256 challengeAmount;            // Amount of collateral token to be locked from the challenger's own balance when challenging actions
-        IStaking staking;                   // Staking pool cache for the collateral token
+        IStaking staking;                   // Staking pool cache for the collateral token -- will never change
     }
 
     struct DisputableInfo {
-        bool activated;                                                     // Whether a Disputable app is activated
-        uint256 nextCollateralRequirementsId;                               // Identification number of the next collateral requirement instance
-        mapping (uint256 => CollateralRequirement) collateralRequirements;  // List of collateral requirements indexed by id
+        bool activated;                                                     // Whether the Disputable app is active
+        uint256 nextCollateralRequirementsId;                               // Identification number of the next collateral requirement
+        mapping (uint256 => CollateralRequirement) collateralRequirements;  // List of collateral requirements indexed by ID
     }
 
-    IStakingFactory public stakingFactory;                          // Staking factory to be used for the collateral staking pools
+    IStakingFactory public stakingFactory;                           // Staking factory, for finding each collateral token's staking pool
 
     uint256 private nextSettingId;
-    mapping (uint256 => Setting) private settings;                  // List of historic settings indexed by ID
-    mapping (address => uint256) private lastSettingSignedBy;       // List of last setting signed by user
-    mapping (address => DisputableInfo) private disputableInfos;    // Mapping of disputable address => disputable infos
+    mapping (uint256 => Setting) private settings;                  // List of historic agreement settings indexed by ID
+    mapping (address => uint256) private lastSettingSignedBy;       // Mapping of address => last agreement setting signed
+    mapping (address => DisputableInfo) private disputableInfos;    // Mapping of Disputable app => disputable infos
```

Comment:
Not sure if this is clearer to be honest; I find it a bit confusing that the "index" is at the end rather than in the same order (aside: this was initially confusing me with DisputableVoting's `representatives`—which address was which?)

Reply:
I'm okay then 👍